scripts/cli.ts

```diff
@@ -0,0 +1,148 @@
+#! /usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { resolve, join } = require('path')
+const {
+  appendFileSync,
+  copyFileSync,
+  existsSync,
+  mkdirSync,
+  readFileSync,
+  rmSync,
+  writeFileSync,
+} = require('fs')
+const { readdir, unlink } = require('fs').promises
+const applyDesignTokens = require('./applyDesignTokens')
+
+const args = process.argv.slice(2)
+
+async function getFiles(dir) {
+  const dirents = await readdir(dir, { withFileTypes: true })
+  const files = await Promise.all(
+    dirents.map((dirents) => {
+      const res = resolve(dir, dirents.name)
+      return dirents.isDirectory() ? getFiles(res) : res
+    })
+  )
+  return files.flat()
+}
+
+async function applyTokens(options: {
+  figmaFileURL: string
+  outputPath: string
+}) {
+  // 1. Create temporary directory.
+  const tmpDir = './liquid_tmp'
+  if (!existsSync(tmpDir)) {
+    mkdirSync(tmpDir, { recursive: true })
+    writeFileSync(join(tmpDir, '.gitignore'), '*', 'utf8')
+  }
+
+  // 2. Fetch design tokens, update CSS and create token file
+  //    inside temp directory.
+  await applyDesignTokens(options.figmaFileURL)
+
+  // 3. Concatenate all CSS files to a liquid.globals.css file.
+  //    Put the file either into a given output directory or
+  //    the current working directory.
+  const globalsCSSPath = join(options.outputPath, './liquid.globals.css')
+  if (existsSync(globalsCSSPath)) {
+    await unlink(globalsCSSPath)
+  }
+  ;(await getFiles(tmpDir)).forEach((file: string) => {
+    if (file.endsWith('.css')) {
+      appendFileSync(globalsCSSPath, readFileSync(file).toString())
+    }
+  })
+
+  // 4. Copy the design-tokens.json file into the current working directory.
+  //    The tailwindPreset.js module will look for the file in that directory.
+  copyFileSync(
```

Comment:
Optional suggestion.

```suggestion
  // 4. Move the design-tokens.json file into the current working directory.
  //    The tailwindPreset.js module will look for the file in that directory.
  renameSync(
```

Reply:
I'll keep it as it is. The directory is removed anyway at the end.